Utilities/Dox/PythonScripts/WebPageGenerator.py

```diff
@@ -1074,6 +1073,7 @@ def generateIndividualGlobalPage(self):
                                 if len(pdfEntryRow) == 8:
                                   pdfEntryList.append(pdfEntryRow)
                                   pdfEntryRow = []
+                            indexList.append("Found Entries")
```

Comment:
I don't understand what this changes.  It seems to just put the appending after the loop instead of before it.

And, running it didn't seem to update as expected.  lobal_XlZBKDE1.html has no "Found Entries" accordion but does have it in the navigation bar.

Reply:
PTAL, it was missing the 'if' statement!